docs/env_vars.md

```diff
@@ -0,0 +1,17 @@
+wlroots reads these environment variables
+
+wlroots specific
+----------------
+* *WLR_DRM_DEVICES*: specifies the DRM device instead of auto probing it
+* *WLR_DRM_NO_ATOMIC*: set to 1 to use legacy DRM interface instead of atomic mode setting
+* *WLR_LIBINPUT_NO_DEVICES*: set to 1 to not fail without any inpout devices
+* *WLR_WL_OUTPUTS*: when using the wayland backend specifies the number of outputs
+* *WLR_X11_OUTPUTS*: when using the X11 backend specifies the number of outputs
+
+generic
+-------
+* *DISPLAY*: if set probe X11 backend in *wlr_backend_autocreate*
+* *WAYLAND_DISPLAY*, *_WAYLAND_DISPLAY*, *WAYLAND_SOCKET*: if set probe Wayland backend in *wlr_backend_autocreate*
+* *XCURSOR_PATH*: path to xcursor library
+* *XDG_RUNTIME_DIR*: used to store non persistent temporary files
+* *XKB_DEFAULT_RULES*, *XKB_DEFAULT_MODEL*, *XKB_DEFAULT_LAYOUT*, *XKB_DEFAULT_VARIANT*, *XKB_DEFAULT_OPTIONS*: xkb setup
```

Comment:
This is used by rootston and the examples, this should be written down (as in: don't expect them to work in any wlroots-based compositor)

Reply:
Oh, I misgrepped that one. I've added a rootston specific section (I think we can ignore the examples for now since anybody looking into these is likely to study the sources).